### ISSUE TITLE
Aria-live and aria-busy attributes missing on loader

### ISSUE THREAD

**Issue by haydencordeiro** [ankurk91/vue-loading-overlay#128]
https://dockyard.com/blog/2020/03/02/accessible-loading-indicatorswith-no-extra-elements

**Comment by ankurk91** [comment]
https://github.com/ankurk91/vue-loading-overlay/blob/6bf02cbd50735442d964eccadac016771078907c/src/js/Component.vue#L7

**Comment by haydencordeiro** [comment]
Can we add the aria-live attribute too? 

I can do it if you have no concerns!

**Comment by haydencordeiro** [comment]
https://github.com/ankurk91/vue-loading-overlay/pull/129

**Comment by haydencordeiro** [comment]
Hey @ankurk91 let me know if you need anything else! I have attached the PR for the change.


Additional references:
"aria-busy lets us indicate that the container is in the process of updating.
aria-live lets us indicate that the element is going to updated. The polite property indicates a background change (this might be a good place to add a prop to indicate if the content is main content, or background content)"
 https://dev.to/antjanus/accessibility-labels-for-content-loading-in-react-hib#:~:text=aria%2Dbusy%20lets,or%20background%20content)
